html/index.html

```diff
@@ -26,13 +27,22 @@
             if (!navigator.geolocation) {
                 status.textContent += "Error (geolocation not supported)";
             } else {
-                console.log('Trying to get location')
+                console.log('Trying to get location');
                 navigator.geolocation.getCurrentPosition(success, error);
             }
-
         }
-
         geoGetCurrentLocation();
+
+        $(document).ready(function(){
+            $(":submit").click(function(){
+                var params = {
+                    searchWord: $("input[name=searchWord]").val()
+                };
+                $.post("/outlets", params, function(res, status) {
```

Comment:
I think before we complete the search function, maybe we use the GET /outlets first? So at least we can show the results first before thinking about parsing the search string.

Reply:
Okay, will try that out now. Thanks Mr Cheng!